forumMessages/models.py

```diff
@@ -6,6 +6,16 @@
 class Category(models.Model):
     name = models.TextField(max_length=50)
     # Category.id field auto-generated by Django, will not be used
+    
+    @classmethod
+    def addCategory(self, categoryName):
+        self.categories.add(Category.objects.filter(name=categoryName))
```

Comment:
what does this do?
There is no categories attribute in the Category class

Reply:
The method addCategory is under Message class, not Category class.
What it does (supposedly, at least...) is add  a Category object to the categories attribute of the Message instance (self).

Reply:
I see the problem. This got the other way around, probably committed in the wrong order and the method jumped back to Category class. I'll fix it.